eternalcore-core/src/main/java/com/eternalcode/core/feature/home/homeadmin/PlayerHomeMultiArgument.java

```diff
@@ -58,50 +63,58 @@ public ParseResult<PlayerHomeEntry> parse(
     ) {
         Viewer viewer = this.viewerService.any(invocation.sender());
 
-        NoticeBroadcast offlinePlayer = this.noticeService.create()
-            .notice(translation -> translation.argument().offlinePlayer())
-            .viewer(viewer);
-
-        if (!rawInput.hasNext()) {
+        if (rawInput.size() < 2) {
             NoticeBroadcast missingPlayerName = this.noticeService.create()
-                .notice(translation -> translation.argument().missingPlayerName())
+                .notice(translation -> translation.home().missingArgument())
                 .viewer(viewer);
+
             return ParseResult.failure(missingPlayerName);
         }
 
-        String playerName = rawInput.next();
-        Player player = this.server.getPlayer(playerName);
+        Optional<User> optionalUser = this.userManager.getUser(rawInput.next());
+
+        if (optionalUser.isEmpty()) {
```

Comment:
can we add fallback for user by extracting user from bukkit (Server#getOfflinePlayer) You should do this action in ParseResult.async 
and then .map